README.md

```diff
@@ -109,24 +119,58 @@ and then restart the containers with:
 ./gradlew :apix-docker:docker-51:composeUp --info
 ```
 
-### Generate development guide 
-The developer guide is a PDF intended for integrators of Alfred API at both customers
-and Xenit.
 
-To generate it execute:
-```bash
-cd developer-documentation
-./generate-doc
-```
-The resulting PDF is written in the same directory.
-
-## Common problems
-* Look in the Alfresco log.
-* When calling Alfred API webscript from inside an integration JUnit test, you must be aware of 
-  transaction effects.
-    * A webscript runs in a separate independent transaction when called over http from an integration 
-      test. Therefore, to have the webscript see changes made in your test, before calling over HTTP, 
-      you have to put in a new transaction. Similarly to see the changes from the webscript in your 
-      JUnit test, create a new transaction after calling HTTP. 
-    * Warning: Do not forget to set `requiresNew` on the transaction in Alfresco as the JUnit test by 
-      default already runs inside a transaction.
+### REST HTTP result codes
```

Comment:
I'm still debating whether this should go in user (web/PDF) or contributor guide (Github). I'm leaning towards user, since they want know what errors mean. In that case we can leave this out here, and link to the website.